src/test/java/org/broadinstitute/hellbender/tools/walkers/sv/SVClusterIntegrationTest.java

```diff
@@ -56,7 +56,7 @@ public void testDefragmentation() {
         Assert.assertEquals(header.getSampleNamesInOrder(), inputHeader.getSampleNamesInOrder());
         Assert.assertEquals(header.getSequenceDictionary().size(), inputHeader.getSequenceDictionary().size());
 
-        Assert.assertEquals(records.size(), 338);
+        Assert.assertEquals(records.size(), 408);
```

Comment:
What caused this change?

Reply:
Interestingly, this is a bug in master. Due to the use of `TreeSet` with a comparator on genomic position, clustered variants with the same start position were getting thrown out.